he/psi_he.ts

```diff
@@ -2584,7 +2584,7 @@ Proceed?</source>
         <location filename="../psi/src/disco.ui" line="28"/>
         <source>ToolBar will be placed here</source>
         <comment>Don&apos;t translate this string</comment>
-        <translation type="unfinished"></translation>
+        <translation>סרגל הכלים ימוקם כאן</translation>
```

Comment:
Comment: Don't translate this string

<comment>Don&apos;t translate this string</comment>